src/app/WriteHandler.h

```diff
@@ -200,10 +200,8 @@ class WriteHandler : public Messaging::ExchangeDelegate
     Optional<ConcreteAttributePath> mProcessingAttributePath;
     Optional<AttributeAccessToken> mACLCheckCache = NullOptional;
 
-#if CHIP_CONFIG_USE_DATA_MODEL_INTERFACE
     DataModel::Provider * mDataModelProvider = nullptr;
     std::optional<ConcreteAttributePath> mLastSuccessfullyWrittenPath;
```

Comment:
Followup: can we have mProcessingAttributePath and mLastSuccessfullyWrittenPath set at the same time?  If not there is some possible savings here...

Reply:
Added to #36327